### PR TITLE
fix(cmd-tables): Call terminate client after docs generation

### PR DESCRIPTION
**Pull request by erezrokah**
#### Summary

Follow up to https://github.com/cloudquery/cloudquery/pull/8803

### CODE REVIEW

**Review by hermanschaaf — APPROVED** [review]
💯

**Comment by erezrokah** [comment]
Not sure why the tests are hanging in CI, looking into it

File: cli/cmd/tables.go [hunk 1]
```diff
@@ -74,6 +74,11 @@ func tables(cmd *cobra.Command, args []string) error {
 		if err != nil {
 			return fmt.Errorf("failed to generate docs. Error: %w", err)
 		}
+
+		err = sourceClient.Terminate()
+		if err != nil {
+			fmt.Println("Failed to terminate source client. Error: ", err)
```
Comment (erezrokah) [inline cli/cmd/tables.go:80]: I think we can keep generating docs for other plugins when this happens. No reason to exit early

**Comment by erezrokah** [comment]
@hermanschaaf This ended up being a bug fix to the hang (not sure why it happens only when running `go test ./..`), so I requested another review

**Review by hermanschaaf — APPROVED** [review]
Makes sense 👍

**Pull request merged**